### PR TITLE
feat(teams-limit): add upgrade CTA to the meeting sidebar if teams limit is exceeded

### PR DESCRIPTION
**Pull request by igorlesnenko**
image

### CODE REVIEW

File: packages/client/components/NewMeetingSidebarUpgradeBlock.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+const NewMeetingSidebarUpgradeBlock = (props: Props) => {
+  const {onClick, orgId} = props
+  const history = useHistory()
+  return (
+    <RootBlock>
+      <Container>
+        <Heading>🎉 We’re glad you love Parabol!</Heading>
+
+        <Description>
+          To make sure you don’t lose access, make sure to upgrade to the Team plan so you can have
```
Comment (nickoferrall) [inline packages/client/components/NewMeetingSidebarUpgradeBlock.tsx:54]: +1 It might be helpful to explicitly state why they might lose access. While the end of the sentence explains that a benefit of upgrading is getting "as many teams as you need", it's not super clear to me why the user might lose access. Perhaps: "You've exceeded the two-team limit. To make sure you don't lose access, upgrade to the Team plan so you can have as many teams as you need."

I realise that this sentence was included in the designs, so feel free to ignore. Curious to get your thoughts though

File: packages/client/components/NewMeetingSidebarUpgradeBlock.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+          as many teams as you need.
+        </Description>
+
+        <Action>
+          <UpgradeButton
+            onClick={() => {
```
Comment (nickoferrall) [inline packages/client/components/NewMeetingSidebarUpgradeBlock.tsx:60]: +1 teeny nit: as this function is doing two things, I'd lean towards creating a `handleClick` function and using it here to make the JSX more readable

File: packages/client/components/NewMeetingSidebarUpgradeBlock.tsx (hunk 1 above)
Comment (igorlesnenko) [reply packages/client/components/NewMeetingSidebarUpgradeBlock.tsx]: Make sense. I updated the copy. 
cc @acressall 

image

File: packages/client/components/NewMeetingSidebarUpgradeBlock.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  onClick?: () => void
+  orgId: string
+}
+
+const NewMeetingSidebarUpgradeBlock = (props: Props) => {
+  const {onClick, orgId} = props
+  const history = useHistory()
+
+  const handleClick = () => {
+    onClick && onClick()
```
Comment (Dschoordsch) [inline packages/client/components/NewMeetingSidebarUpgradeBlock.tsx:50]: +1 you could shorten this to `onClick?.()`

**Review by Dschoordsch — CHANGES_REQUESTED** [review]
Code looks good. Only on tiny screens (horizontal phone) the meeting stages can't be reached at all anymore. The current scrollable area only includes the stages, but I think also the facilitator needs to be in the same area to make it work.
Screenshot from 2022-12-08 18-20-11
Screenshot from 2022-12-08 18-18-16

**Comment by igorlesnenko** [comment]
@Dschoordsch if I made scrollable area bigger, it will work, but it will be tiny anyway. Maybe add close button on mobile only? 
image

**Comment by igorlesnenko** [comment]
@Dschoordsch added close button here is the demo
https://www.loom.com/share/404cb9784b0a4eb0a8898e462297643b

**Review by Dschoordsch — APPROVED** [review]
